driver/src/main/java/org/neo4j/driver/NotificationCategory.java

```diff
@@ -17,50 +17,48 @@
 package org.neo4j.driver;
 
 import java.io.Serializable;
-import org.neo4j.driver.internal.InternalNotificationCategory;
-import org.neo4j.driver.internal.InternalNotificationCategory.Type;
 
 /**
  * Notification category.
  *
  * @since 5.7
  */
-public sealed interface NotificationCategory extends Serializable permits InternalNotificationCategory {
+public sealed interface NotificationCategory extends Serializable permits NotificationClassification {
```

Comment:
I don't like the interface permits but also depends on implementation (here enum reference). But on the other hand this should get deprecated and later just the `NotificationClassification` stays, or?

Reply:
Exactly. The `NotificationClassification` should replace the `NotificationCategory`.